app/javascript/tests/patient/common_exposure_cohorts/CommonExposureCohortsTable.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CommonExposureCohortTable from '../../../components/patient/common_exposure_cohorts/CommonExposureCohortsTable';
+
+import { mockCommonExposureCohort1, mockCommonExposureCohort2 } from '../../mocks/mockCommonExposureCohorts';
+
+const onEditCohortMock = jest.fn();
+const onDeleteCohortMock = jest.fn();
+
+const cohorts = [mockCommonExposureCohort1, mockCommonExposureCohort2];
+
+function getWrapper(isEditable) {
+  return shallow(<CommonExposureCohortTable common_exposure_cohorts={cohorts} isEditable={isEditable} onEditCohort={onEditCohortMock} onDeleteCohort={onDeleteCohortMock} />);
+}
+
+describe('CommonExposureCohortsTable', () => {
+  it('Properly renders common exposure cohorts', () => {
+    const wrapper = getWrapper(false);
+    cohorts.forEach((cohort, index) => {
+      expect(wrapper.find(`#common-exposure-cohort-edit-button-${index}`).exists()).toBe(false);
```

Comment:
can we make these two tests more specific and check the row?
```suggestion
      expect(wrapper.find('tr').at(index).find(`#common-exposure-cohort-edit-button-${index}`).exists()).toBe(false);
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/913b2d76b4971c4f97c842472610dff60f97b69c